### PR TITLE
Task branch: pagination added, redundant method in task controller deleted

### PR DESCRIPTION
**Pull request by Sofia-Ingl**

### CODE REVIEW

**Comment by IP-13** [comment]
|Overall Project|75.83%|:green_apple:|
|:-|:-|:-:|
|Files changed|100%|:green_apple:|
<br>

|File|Coverage||
|:-|:-|:-:|
|[TaskController.java](https://github.com/itmo-events-app/event-app/blob/fb625627d84a7b29f2f2aad3a8771439ee979020/src%2Fmain%2Fjava%2Forg%2Fitmo%2Feventapp%2Fmain%2Fcontroller%2FTaskController.java)|90.61%|:green_apple:|
|[TaskService.java](https://github.com/itmo-events-app/event-app/blob/fb625627d84a7b29f2f2aad3a8771439ee979020/src%2Fmain%2Fjava%2Forg%2Fitmo%2Feventapp%2Fmain%2Fservice%2FTaskService.java)|88.61%|:green_apple:|

File: src/main/java/org/itmo/eventapp/main/controller/TaskController.java [hunk 1]
```diff
@@ -164,77 +169,121 @@ public ResponseEntity<List<TaskResponse>> taskListCopy(
...
+            @Min(value = 1, message = "Параметр assignerId не может быть меньше 1!")
+            @RequestParam(required = false)
+            @Parameter(name = "assignerId", description = "ID Создателя задачи", example = "13")
+            Integer assignerId,
+            @Valid @RequestParam(required = false)
+            @Parameter(name = "taskStatus", description = "Статус задачи")
+            TaskStatus taskStatus,
+            @RequestParam(required = false)
+            @Valid @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME)
+            @Parameter(name = "deadlineLowerLimit", description = "Мягкий дедлайн задачи")
```
Comment (Darklol) [inline src/main/java/org/itmo/eventapp/main/controller/TaskController.java:251]: Не замечал этого раньше, что за "жёсткий" и "мягкий" дедлайн?

Comment (Sofia-Ingl) [reply src/main/java/org/itmo/eventapp/main/controller/TaskController.java:251]: Хз... Это не я написала ахаххаха. Тоже не замечала. Ну вообще lower limit - просто нижняя граница фильтрации, там выполняется gt & lt, причем включительно

Comment (Sofia-Ingl) [reply src/main/java/org/itmo/eventapp/main/controller/TaskController.java:251]: Ща исправлю пояснения к параметрам

**Comment by IP-13** [comment]
|Overall Project|76.11%|:green_apple:|
|:-|:-|:-:|
|Files changed|100%|:green_apple:|
<br>

|File|Coverage||
|:-|:-|:-:|
|[TaskController.java](https://github.com/itmo-events-app/event-app/blob/bffcf54c98a52182d108e96db597c344ad1de8c6/src%2Fmain%2Fjava%2Forg%2Fitmo%2Feventapp%2Fmain%2Fcontroller%2FTaskController.java)|90.61%|:green_apple:|
|[TaskService.java](https://github.com/itmo-events-app/event-app/blob/bffcf54c98a52182d108e96db597c344ad1de8c6/src%2Fmain%2Fjava%2Forg%2Fitmo%2Feventapp%2Fmain%2Fservice%2FTaskService.java)|88.61%|:green_apple:|

File: src/main/java/org/itmo/eventapp/main/controller/TaskController.java [hunk 2]
```diff
@@ -164,77 +169,121 @@ public ResponseEntity<List<TaskResponse>> taskListCopy(
...
+            @PathVariable
+            @Parameter(name = "eventId", description = "ID мероприятия", example = "1")
+            Integer eventId,
+            @Min(value = 1, message = "Параметр assigneeId не может быть меньше 1!")
+            @RequestParam(required = false)
+            @Parameter(name = "assigneeId", description = "ID Исполнителя задачи", example = "123")
+            Integer assigneeId,
+            @Min(value = 1, message = "Параметр assignerId не может быть меньше 1!")
+            @RequestParam(required = false)
+            @Parameter(name = "assignerId", description = "ID Создателя задачи", example = "13")
```
Comment (HackMemory) [inline src/main/java/org/itmo/eventapp/main/controller/TaskController.java:181]: Если assignerId не является обязательным, то какой userId в итоге присвоится, если его не указать и personalTasksGet = False?

**Review by IP-13 — APPROVED** [review]

**Comment by IP-13** [comment]
|Overall Project|76.29%|:green_apple:|
|:-|:-|:-:|
|Files changed|100%|:green_apple:|
<br>

|File|Coverage||
|:-|:-|:-:|
|[TaskController.java](https://github.com/itmo-events-app/event-app/blob/ce33275cd3641d13b03d9d2293776f4b733e86e3/src%2Fmain%2Fjava%2Forg%2Fitmo%2Feventapp%2Fmain%2Fcontroller%2FTaskController.java)|90.61%|:green_apple:|
|[TaskService.java](https://github.com/itmo-events-app/event-app/blob/ce33275cd3641d13b03d9d2293776f4b733e86e3/src%2Fmain%2Fjava%2Forg%2Fitmo%2Feventapp%2Fmain%2Fservice%2FTaskService.java)|88.61%|:green_apple:|

**Pull request merged**